### DISCUSSION TITLE
启动器病毒出来了

### DISCUSSION THREAD

**Discussion by Dapen114** [Meloong-Git/PCL#2426] [讨论]
有个人伪装启动器，包括您的启动器，HMCL,baka XL，打开后突然出现弹窗病毒，直到蓝屏后，引导没了
![46d7e352ed039b5d340dda0d1a63f49](https://github.com/Hex-Dragon/PCL2/assets/138861695/eef44d7f-7d39-4cd5-866d-802e05a67577)
![01bd198d96b1debf5285a9c5d193ec4](https://github.com/Hex-Dragon/PCL2/assets/138861695/803f6e15-2ee8-4982-b3cd-097219b7c0b2)

**Comment by zkitefly** [comment]
所以呢（

是要做些什么吗

**Comment by XiaoLLiu617** [comment]
放心，不必担心

**Comment by Chiloven945** [comment]
明显整活
这种为了流量无视道德底线带风向的人，用不要脸形容恐怕就也是最委婉的了

**Comment by SuoHouDaiShi** [comment]
（Tip）本来写了比较长的一段话，想到要是被定为爆粗口可就不好了，所以我就**一张图**：
![QQ图片20230808163142](https://github.com/Hex-Dragon/PCL2/assets/119282947/f60c414f-56f2-4f6f-9023-d13015fcf33b)

**Reply by Chiloven945** [reply]
中肯的

**Comment by MinecraftYJQ** [comment]
大家好，我是病毒原作者。抱歉让大家生气了（ 
首先，我做这个病毒只是为了整蛊同学，没想到弄起了这么大的影响。所以，我对龙猫说:对不起。
然后柚子让我改的封面，我晚一点会改。
对不起。
然后我也没什么东西弄了，不如大家给点意见awa？
